src/components/AddressExplorerLink/index.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import { getAddress, isAddress } from '@ethersproject/address';
 
-type ChainID = 1 | 5 | 42170 | 421613 | 42161;
+type ChainID = 1 | 11155111 | 42170 | 421614 | 42161;
 
 const chainIDToExplorerUrlRoot: {
   [chainId in ChainID]: string;
 } = {
   1: 'https://etherscan.io/address',
-  5: 'https://goerli.etherscan.io/address',
+  11155111: 'https://sepolia.etherscan.io',
   42161: 'https://arbiscan.io/address',
   42170: 'https://nova.arbiscan.io/address',
   
-  421613: 'https://goerli.arbiscan.io/address',
+  421614: 'https://sepolia.arbiscan.io',
```

Comment:
```suggestion
  421614: 'https://sepolia.arbiscan.io/address',
```

links are failing because they need the `/address/` path